src/RefractAPI.cc

```diff
@@ -187,26 +187,18 @@ namespace drafter {
     refract::IElement* ParameterToRefract(const NodeInfo<snowcrash::Parameter>& parameter, ConversionContext& context)
     {
         refract::MemberElement* element = new refract::MemberElement;
-        refract::IElement *value = NULL;
-
-        // Parameter type, exampleValue, defaultValue, values
-        if (parameter.node->type == "boolean") {
-            value = ExtractParameter<bool>(parameter, context);
-        }
-        else if (parameter.node->type == "number") {
-            value = ExtractParameter<double>(parameter, context);
-        }
-        else {
-            value = ExtractParameter<std::string>(parameter, context);
-        }
-
+        refract::IElement *value = ExtractParameter<std::string>(parameter, context);
```

Comment:
It is no more required to be `template`.
Please remove template behavior `ExtractParameter()` - and use strictly `std::string` inside of function